### PR TITLE
added route and 404

### PR DESCRIPTION
**Pull request by LokeshKumar-Dev**
- [x] Added 404 not found page
- [x] Added All and default route logic

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -32,7 +33,39 @@
...
+.btn {
+  padding: .5rem 1rem;
+  border-radius: 10px;
+  color: white;
+  background-color: rgb(11, 60, 138) !important;
+}
+
+.btn:hover {
+  background-color: rgba(21, 68, 143, 0.761) !important;
+}
```
Comment (ravijyoti3) [inline src/App.css]: @LokeshKumar-Dev please use tailwind css for any styling.

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,41 @@
...
-        </Switch>
-    </BrowserRouter>
+    <Layout className="body">
+      <BrowserRouter>
+        <SideBar />
+        {/* <ToastContainer /> */}
+        <Content
+          style={{
+            padding: "30px 25px",
+            minHeight: 280,
```
Comment (ravijyoti3) [inline src/App.js]: @LokeshKumar-Dev Use tailwind css only.

File: src/Components/Article/SideBar.jsx [hunk 3]
```diff
@@ -31,38 +34,51 @@ const SideBar = () => {
...
+           border-2 rounded-full ${!open ? "rotate-[0]" : "rotate-180"}`}
+          // style={{ transform: "rotate(180deg)" }}
           onClick={() => setOpen(!open)}
         />
-        <div className="flex gap-x-4 items-center">
+        <div
+          className="flex gap-x-4 items-center"
+          style={{
+            height: "30px",
+          }}
```
Comment (ravijyoti3) [inline src/Components/Article/SideBar.jsx]: @LokeshKumar-Dev

File: src/Components/Article/index.jsx [hunk 4]
```diff
@@ -4,10 +4,13 @@ import SideBar from "./SideBar";
...
 import ArticleSection from "./ArticleSection";
-import NotFound from "../common/NotFound";
 
 const index = () => {
-  return <NotFound />;
+  return (
+    <>
+      <ArticleSection />
+    </>
+  );
```
Comment (ravijyoti3) [inline src/Components/Article/index.jsx]: @LokeshKumar-Dev These fragments are useles, you can directly use `<ArticleSection/>`
```suggestion
  return (
      <ArticleSection />
  );
```

**Review by ravijyoti3 — CHANGES_REQUESTED** [review]
@LokeshKumar-Dev  Please make the changes suggested

**Comment by LokeshKumar-Dev** [comment]
All Done!!!
[https://www.loom.com/share/75bbe54f1a0e4fe888e3830e69571a0d](url)

**Comment by ravijyoti3** [comment]
Always create pull request to your own repo

@LokeshKumar-Dev

**Pull request closed without merging**